### PR TITLE
Add 'make install' to readme

### LINKED ISSUE #900: Readme.md --> add "make install"
The [installation instructions in Readme.md](https://github.com/symengine/symengine#installation) should probably contain a 

`make install`

so people not familiar with building do not get stuck.

### PR DESCRIPTION
**Pull request by CodeMaxx**
Fixes #900
<br/>I think this can be merged without any effect on the clang format PR.
@isuruf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ For RPM based systems (Fedora etc.):
 
     yum install cmake gmp-devel
 
-Install SymEngine:
+Build SymEngine:
 
     cmake .
     make
```
Comment (isuruf) [inline README.md]: You can add it here

File: README.md [hunk 2]
```diff
@@ -49,6 +49,12 @@ Run tests:
 
     ctest
 
+Install SymEngine:
+```
+sudo make install
```
Comment (isuruf) [inline README.md]: Don't use `sudo`

**Comment by isuruf** [comment]
For PRs with no code changes, you can add [skip ci] to the commit message to skip travis and appveyor

**Comment by CodeMaxx** [comment]
Cool! I had been wondering if there was some way to do that.

File: README.md [hunk 3]
```diff
@@ -41,9 +41,11 @@ Install SymEngine:
 
     cmake .
     make
+    make install
 
 This will configure and build SymEngine in the default Release mode with all
-code and compiler optimizations on.
+code and compiler optimizations on and then install it on your system.
+SymEngine can then be imported in your programs.
```
Comment (isuruf) [inline README.md]: This would incorrectly imply that you don't have to link with SymEngine after installing.

Comment (CodeMaxx) [inline README.md]: Should I add `Note that you will have to link SymEngine during compilation` or should I remove this line?

Comment (isuruf) [inline README.md]: Removing the line would be better

Comment (CodeMaxx) [inline README.md]: Done

**Pull request merged**